### PR TITLE
fix: Long text does not display in pivot tables in data visualizer app [2.41-DHIS2-14188]

### PR DESCRIPTION
**Pull request by d-bernat**
Please see: https://dhis2.atlassian.net/browse/DHIS2-14188

### CODE REVIEW

File: dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    public void setup()
+    {
+        analyticsActions = new RestApiActions( "analytics" );
+    }
+
+    @Test
+    public void testAnalyticsGetWithTextDataElementAggregationTypeNone()
+    {
+        // Given
+        QueryParamsBuilder params = new QueryParamsBuilder()
```
Comment (maikelarabori) [inline dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java:62]: Maybe is safer to add a sorting parameter to the URL, to ensure the same order of results?

File: dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+        validateRow( response, 1,
+            List.of( "M3xtLkYBlKI.fyjPqlHE7Dn", "202107", "Some insecticide resistance" ) );
+    }
+
+    @Test
+    public void testAnalyticsGetWithLongTextDataElementAggregationTypeSum()
+    {
+        // Given
+        QueryParamsBuilder params = new QueryParamsBuilder()
```
Comment (maikelarabori) [inline dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java:88]: Same here

**Review by maikelarabori — APPROVED** [review]
Looks good Dusan :+1: just two minor comments on tests.

File: dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java (hunk 1 above)
Comment (d-bernat) [reply dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java:62]: Done

File: dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java (hunk 2 above)
Comment (d-bernat) [reply dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/analytics/AnalyticsQueryTest.java:89]: Done

**Pull request merged**